CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/CameraPreviewViewController.swift

```diff
@@ -38,6 +38,14 @@ final class CameraPreviewViewController: UIViewController {
         spinner.hidesWhenStopped = true
         return spinner
     }()
+
+    lazy var cameraFrameView: UIView = {
+        let imageView = UIView()
+        imageView.layer.borderColor = UIColor.white.cgColor
```

Comment:
[Suggestion] Please use the color here for both modes: light and dark

Reply:
You mean to have the same color for both modes, or to have white/black regarding light/dark mode?

Reply:
We've agreed to add a frame color to the color palette.